airflow/api_fastapi/execution_api/routes/xcoms.py

```diff
@@ -70,7 +70,7 @@ async def xcom_query(
         map_indexes=map_index,
         session=session,
     )
-    return query.with_entities(BaseXCom.value)
+    return query
```

Comment:
Curious, why this change?

Reply:
Because for the `get` path we _want_ to go via the orm_deserialize_path, but by doing `SELECT value FROM xcom` (which is what `with_entities()` does) we don't get that.

Reply:
Cool thanks, resolving